docs/core/windows-prerequisites.md

```diff
@@ -4,54 +4,89 @@ description: Learn what dependencies you need on your Windows machine to develop
+ 1. [Download and install Visual Studio 2017 version 15.3.0 or higher](/visualstudio/install/install-visual-studio) with the **.NET Core cross-platform development** workload (in the **Other Toolsets** section) selected.
+![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs-15-3-workloads.jpg)
 
-To verify that you're running the latest version of Visual Studio 2017, do the following:
+After the **.NET Core cross-platform development** toolset is installed, Visual Studio 2017 uses .NET Core 1.x by default. Install the .NET Core 2.x SDK to get .NET Core 2.x support in Visual Studio 2017.
 
- * On the **Help** menu, choose **About Microsoft Visual Studio**.
- * In the **About Microsoft Visual Studio** dialog, the version number should be 15.0.26228.4 or higher.
+ 2. Install the [.NET Core 2.x SDK](https://www.microsoft.com/net/download/core).
+ 3. Retarget existing or new .NET Core 1.x projects to .NET Core 2.x using the following instructions:
+   * On the **Project** menu, Choose **Properties**. 
+   * In the **Target framework** selection menu, set the value to **.NET Core 2.0**.
 
-You can read more about the changes in Visual Studio 2017 in the [release notes](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes).
+![Screenshot of Visual Studio 2017 Application Project Property with the ".NET Core 2.0" Target framework menu item selected](./media/windows-prerequisites/Targeting-dotnetCore2.png)
+
+Once the .NET Core 2.x SDK is installed, Visual Studio 2017 uses the .NET Core SDK 2.x by default, and supports the following actions:
+
+  * Open, build, and run existing .NET Core 1.x projects.
+  * Retarget .NET Core 1.x projects to .NET Core 2.x, build, and run.
+  * Create new .NET Core 2.x projects.
+
+# [.NET Core 1.x](#tab/netcore1x)
+To develop .NET Core 1.x apps in Visual Studio, [download and install Visual Studio 2017 RTM (version 15.0.26228.4) or higher](/visualstudio/install/install-visual-studio) with the **".NET Core cross-platform development"** workload (in the **Other Toolsets** section) selected.
+![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs_workloads.jpg)
+> [!IMPORTANT]
+> It's possible to use Visual Studio 2015 for .NET Core 1.x development, but it's not recommended for the following reasons:
+  > * The .NET Core tooling is a preview version, which is not supported.
+  > * The projects are project.json-based, which is deprecated.
+>
+> For more information about the project format changes, see [High-level overview of changes](./tools/cli-msbuild-architecture.md).
+---
+>[!TIP]
+  > To verify your Visual Studio 2017 version:
+>
+     > * On the **Help** menu, choose **About Microsoft Visual Studio**.
+     > * In the **About Microsoft Visual Studio** dialog, verify the version number.
```

Comment:
I like how you moved this to end. However, you need to add the version numbers along with the version names so they can see if they have the minimum required version.